src/ui/Base/RegisteredNamedMethods.ts

```diff
@@ -248,17 +248,26 @@ function getCoveoAnalytics(element: HTMLElement) {
  * service automatically converts white spaces to underscores, and uppercase characters to lowercase characters in key
  * names. Each value must be a simple string. If you do not need to log metadata, you can simply pass an empty JSON
  * ( `{}` ).
+ * @param result The IQueryResult that the custom event is linked to, if any.
```

Comment:
No other params are commented here, so it's not super consistent.

It would have been a good idea to update the JSDOC to include all of them, and include @fbeaudoincoveo in the review process.

Reply:
Actually, the other parameters are documented.

I can review the doc for this new one, though :)

Suggestion:

> The query result that relates to the custom event, if applicable.

(TypeDoc will automatically generate an hyperlink to the IQueryResult interface, so specifying the exact type in the description is redundant).

Reply:
Param documentation updated as @fbeaudoincoveo suggested